JSDemos/Demos/PivotGrid/DrillDown/description.md

```diff
@@ -1 +1,5 @@
-Drill-down support within the PivotGrid component allows you to&nbsp;retrieve the list of&nbsp;records used to&nbsp;calculate a&nbsp;specific summary value. To&nbsp;see drill downs in&nbsp;action, click a&nbsp;summary cell and view the records that formed the basis of&nbsp;the summary value.
\ No newline at end of file
+PivotGrid supports the drill down operation that allows users to retrieve the list of facts used to calculate a specific summary value. 
+
+Use the [createDrillDownDataSource(options)](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Methods/#createDrillDownDataSourceoptions) method to access underlying facts for a summary value. This method returns a [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) instance that contains a list of facts that form the basis of the summary value.
+
+This demo shows how to display a list of facts in a [Popup](/Documentation/ApiReference/UI_Components/dxPopup/) that contains a [DataGrid](/Documentation/ApiReference/UI_Components/dxDataGrid). We use the [onCellClick](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#onCellClick) handler to retrieve a cell's data and call the **createDrillDownDataSource(options)** within this handler. Then we pass the created **DataSource** to the **DataGrid**'s [dataSource](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#dataSource) property. As a result, you can click a pivot grid cell to display a popup. This popup shows a list of facts used to calculate the summary value from the clicked cell.
```

Comment:
Can drill-downs be shown in a component other than a Popup?

Reply:
No, I don't think so. There are too many scenarios how users can process this data but I'll ask the team about it.

Reply:
If there were such components, would it be good to list them in the docs?
Shortened the para:
```suggestion
In this demo, we use the [onCellClick](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#onCellClick) handler to create a [DataGrid](/Documentation/ApiReference/UI_Components/dxDataGrid) of facts and display it inside a [Popup](/Documentation/ApiReference/UI_Components/dxPopup/). To do this, we pass the clicked cell's data as an argument for **createDrillDownDataSource** and bind the returned [dataSource](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#dataSource) to the DataGrid.
```

Reply:
> This demo shows how to [do X]

Do not use this intro for custom scenarios. A developer may have a different scenario. Use "This demo does X" or "In this demo, X is done".

> We use the [onCellClick](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#onCellClick) handler to retrieve a cell's data

You haven't told yet that a cell should be clicked. You tell it a couple of sentence after: "As a result, you can click a pivot grid cell to display a popup.".

> Then we pass the created **DataSource** to the **DataGrid**'s [dataSource](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#dataSource) property.

This is about the actual scenario. I think we can remove sentence.

```suggestion
This demo displays a list of facts within a [DataGrid](/Documentation/ApiReference/UI_Components/dxDataGrid) in a [Popup](/Documentation/ApiReference/UI_Components/dxPopup/) window. To open the window, click a pivot grid cell. We use the [onCellClick](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#onCellClick) event handler to retrieve the cell's data and pass it as an argument to the **createDrillDownDataSource(options)** method.
```